arches/app/templates/views/components/search/search-results.htm

```diff
@@ -43,7 +43,7 @@
             </div>
         </div>
         <div class="search-listing-footer">
-            <div style="flex-grow: 1;">
+            <div style="flex-grow: 1; padding-bottom: 10px;">
```

Comment:
This looks fine if wrapping is necessary, but if it isn't it causes the footer to be too talk and the text vertically uncentered.